dbfit-java/build.gradle

```diff
@@ -222,3 +251,114 @@ project('netezza') {
     }
 }
 
+// Aggregated dependencies to include in the POM file when publishing.
+dependencies {
+    compile publicCoreCompileDeps
+    runtime publicHsqldbRuntimeDeps
+    compile publicOracleCompileDeps
+    compile publicMysqlCompileDeps
+    compile publicPostgresCompileDeps
+}
+
+// Remove the default artifact.
```

Comment:
Ok, so `reverse()` is just to get at the last element in the `List` and `find()` finds the first matching element. So both of these mean that this will work only if the last default artefact is a JAR (which it would be for our specific case).

Or shall we just remove all of the default artefacts?